### DISCUSSION TITLE
Windows Package Manager REST Source Reference Implementation 1.1.20211029

### DISCUSSION THREAD

**Discussion by denelon** [microsoft/winget-cli-restsource#86] [General]
This is the first release with binaries for the Windows Package Manager REST API reference implementation. 

The reference implementation was designed to run on Azure. The Windows Package Manager client would work with any conforming REST source regardless of the infrastructure it runs on. 

This release includes preview PowerShell cmdlets to aid in deploying the reference REST source implementation to Azure. 
The [current documentation](https://github.com/microsoft/winget-cli-restsource/tree/main/Tools/PowershellModule/doc) includes instructions for deploying the REST source, and working with manifests.

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/winget-cli-restsource/releases/tag/1.1.20211029'>Windows Package Manager REST Source Reference Implementation 1.1.20211029</a>.</em>

**Comment by doctordns** [comment]
As a PowerShell _user_ I am unclear what to do with this. 

Can you please publish the modules to the PowerShell gallery so PowerShell users like myself can just download them and use them?
Or zip them up and release them here.

Alternatively can you provide directions as to how I would use what is here and be able to test.

**Reply by denelon** [reply]
We're still going through our internal governance processes to publish a module to the PowerShell gallery.

We have [instructions for local use](https://github.com/microsoft/winget-cli-restsource/blob/main/Tools/PowershellModule/doc/new-winget-rest-source-azure.md).
